dist/icon/icon.css

```diff
@@ -155,6 +155,14 @@ svg.icon--audio-off-16 {
   height: 16px;
   width: 16px;
 }
+svg.icon--authenticity-guarantee-16-fit {
```

Comment:
Why do some of the icons use `-fit` postfix? I thought that name was reserved for bespoke/cropped icons?

Reply:
This is from design. They asked to have it.

Reply:
What is the reason for it? What does fit mean?

Reply:
This is because the place it's used the program badges need to be sized a bit different than normal. So any `fit` badges seem to be badges that do not have standard sizes and are used to be placed in specific inline scenarios. In a sense, they "fit" within the paragraph or place they are placed in.

Reply:
From Ryan:

> the -fit postfix is used for traditional icons where the artboard is "fit" to the width of the artwork vs a perfect square. These versions are often used for icon+text pairings to have more accurate spacing between elements for pairs that are used often such as our new program badges. There is also an opportunity to use them for icons that have excess space on the left and right (space tbd). We talked about 3px, but Arthur recommended upping it from there so there are less of them, really just for the extreme examples.
Want to mention that -fit is used for icons that would typically have square artboards. And it's meant to also signal that they are not square.